Other/SelectableCollection.cs

```diff
@@ -0,0 +1,194 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Text;
+using System.Windows;
+
+namespace DolphinDynamicInputTextureCreator.Other
+{
+    /// <summary>
+    /// specify the position in the list to be selected
+    /// </summary>
+    public enum SelectPropertie { Keep = default , First, Last, Previous, Next, Null}   
```

Comment:
This is personal preference but I'd suggest:

`public enum SelectionBehavior`

Reply:
you can also use it to set the selection. I don't find Behavior perfect for this either.